src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -444,8 +444,10 @@ protected static void AddMappedDocument(Workspace workspace, string markup)
                 var text = await document.GetTextAsync(CancellationToken.None);
                 foreach (var (name, spans) in testDocument.AnnotatedSpans)
                 {
+                    Contract.ThrowIfNull(document.FilePath);
+
                     var locationsForName = locations.GetValueOrDefault(name, new List<LSP.Location>());
-                    locationsForName.AddRange(spans.Select(span => ConvertTextSpanWithTextToLocation(span, text, new Uri(document.FilePath))));
```

Comment:
could we possibly block this constructor.  we're def going to make thsi mistake in the future.